src/bci_build/package/__init__.py

```diff
@@ -517,6 +517,11 @@ class BaseContainerImage(abc.ABC):
     #: present
     logo_url: str = ""
 
+    #: Optional build_release setting that will be inserted as the
+    #: ``#!BuildRelease`` magic comment for the build service to ensure that
+    #: containers sort higher across service packs
+    _build_release: int | None = None
```

Comment:
I like this being moved here, but maybe a minor nit to call it `_min_release_counter` or something like that to be more descriptive of its purpose?